### ISSUE TITLE
Radio Button and Radio Button Group - Ensure active user interface components have sufficient contrast - (2036570149)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7811]
### Violation: 
 Ensure active user interface components have sufficient contrast 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/bcb6071f-e824-4648-873e-b56a1fd40ea2)

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 6 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Radio Button and Radio Button Group - Ensure active user interface components have sufficient contrast - (2036570149) 
- Module: Radio Button and Radio Button Group 
**Issue**
The radio button borders with a contrast ratio below 3.00:1 when checking with "Toggle theme" on.

Foreground: #6D6D6D
Background: #353535
Contrast ratio: 2.4:1

**User Impact**
Users with low vision will have difficulty identifying this content.

**Code Reference** 
 ```
<div aria-checked="false" aria-label="Maps" class="container" role="radio" tabindex="-1"><div class="radio"></div></div>
 ``` 
**Suggestion**
Ensure active user interface components have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying components include text field borders, checkmarks for checkboxes, fillings for radio buttons, focus indicators, and icon-only controls.

For borders, the "adjacent color" can be the color that touches the outside of the border or the color that touches the inside of the border. Contrast with both is not required.

Disabled controls that cannot be navigated to with the keyboard are exempt from this requirement.

Refer to the Accessible Color Picker extension or Color Contrast Checker site for assistance: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Duplicate - closing in favor of https://github.com/Esri/calcite-design-system/issues/7822
